test/success/semi-simplicial.prl

```diff
@@ -70,8 +70,7 @@ Thm Pick/succ : [
 ] by [
   lam n, m. auto;
 
-  [ NatArg; EqNatRec; EqNatRec ];
-  [ EqNatRec; NatArg; EqNatRec; EqNatRec ]
+  repeat { NatArg || EqNatRec }
```

Comment:
We still need to prove well-formedness of the definition of `Pick` because we have to unfold `Pick`. This is related to #333.